best_fork/MANIFEST

```diff
@@ -1 +1,2 @@
-brainentropy
\ No newline at end of file
+brainentropy
+nirs-toolbox
```

Comment:
For copyright and distribution matters, we should not fork the nirs-toolbox here.
Instead, let's expect it to accessible in the path and add some check to complain if it is not found where we need it.

A problem is that if you delete nirs-toolbox and then commit in this branch, it will remain in the history.
I think the easiest would be to backup this branch and create a new one with all the changes you've made but where there is no fork of nirs-toolbox.
Then you make a new PR with this new branch and get rid of the old one where the nirs-toolbox has been forked.